Create mysql_configuration_secure_file_priv_empty.md (#56)
percona-platform/portal-doc: docs/advisors/checks/mysql_configuration_secure_file_priv_empty.md

> # mysql_configuration_secure_file_priv_empty

Reviewer: Does this sound OK?
```MySQL secure_file_priv configuration option empty check```

> # mysql_configuration_secure_file_priv_empty

Author: ```suggestion
# MySQL secure_file_priv configuration option empty check
```

## Description

> The option secure_file_priv defaults to NULL, which essentially allows users with FILE privilege to create files at any location where MySQL server has write permission. This is considered less secure and against Percona Best Practices.

Reviewer 2: ```suggestion
The option **secure_file_priv** defaults to NULL, which essentially allows users with FILE privilege to create files at any location where MySQL server has Write permission. This is considered less secure and goes against Percona's Best Practices.
```

## Resolution

> To provide a more secure installation, scope of FILE privilege should be restricted using a secure default value for --secure-file-priv. 

Reviewer 2: ```suggestion
To provide a more secure installation, the scope of FILE privilege should be restricted using a secure default value for **--secure-file-priv**. 
```

> Edit my.cnf to provide secure-file-priv configuration and provide a specific location for users with FILE privileges to create files. This is not a dynamic variable and will need an instance reboot.

Reviewer 2: ```suggestion
Edit **my.cnf** to provide secure-file-priv configuration and provide a specific location for users with FILE privileges to create files. This is not a dynamic variable and will need an instance reboot.
```
